### ISSUE TITLE
Structure

### ISSUE THREAD

**Issue by NJMannai** [dnet/androsphinx#21]
Device: 

- Nokia 7.1, Android 10, TalkBack 13.  
- Pixel 4a, Android 12, TalkBack 13. 

There is no header structure. The titles displayed in the app are not formatted as headings. People who use assistive devices often navigate using a header list, which gives them a quick overview of what's happening on the page. 

Same goes for the instructions on the onboarding, they are not well structured. Lists are missing and this is also important for navigation. 

This is related to WCAG 1.3.1 Info and relations.
